AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -652,4 +653,46 @@ - (MSLogContainer *)createLogContainerWithId:(NSString *)batchId {
   return logContainer;
 }
 
+ /* Commented this test because it’s causing other tests in this class to fail.
```

Comment:
@clpolet No, this is a new test for covering the fix, but the test generates flaky behavior because we mock msLogger which used in other tests. This test is not in develop branch yet

Reply:
If it doesn't interfere other tests in this branch, we should check it in.

Reply:
We need at least to create a bug to fix this.

Reply:
I still don't understand why we can't just fix it? Is it too much work?

Reply:
@dtrabo Can you tell me details of commenting this out? This test is written to cover hiding app secrets in logs, if it causes any troubles then you should fix that in the PR.

Reply:
Yes, it’s a cause of troubles. We mock MSLogger in this test when other tests start they use this mocked MSLogger. But when this new test finishes it stop mocking MSLgger and other tests lose an instance of MSLogger and they fail. That’s the reason for the troubles and we don’t know how to fix it.

Reply:
I rewrite the test. It should be good.